test/jdk/java/foreign/TestUpcallException.java

```diff
@@ -33,71 +33,26 @@
  *   TestUpcallException
  */
 
-import jdk.test.lib.Utils;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
-import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.nio.file.Paths;
-import java.util.List;
 
-import static org.testng.Assert.assertFalse;
-import static org.testng.Assert.assertNotEquals;
-import static org.testng.Assert.assertTrue;
+public class TestUpcallException extends UpcallTestHelper {
```

Comment:
I refactored this class to use a common utility method in the super class, which is also used by the new test